HW_1/task_1.py

```diff
@@ -0,0 +1,12 @@
+'''Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0)'''
+
+n = int(input("Введите 3х значное число: "))
+sum = 0
+while n > 0:
+    x = n % 10
+    sum = sum + x
+    n = n // 10
```

Comment:
Тут тоже можно записать короче.
```suggestion
    n //= 10
```